src/Core/Content/Seo/SeoUrlGenerator.php

```diff
@@ -116,11 +132,15 @@ private function generateUrls(SeoUrlRouteInterface $seoUrlRoute, SeoUrlRouteConf
         }
     }
 
-    private function getSeoPathInfo(SeoUrlMapping $mapping, SeoUrlRouteConfig $config): ?string
+    private function getSeoPathInfo(Environment $twig, SeoUrlMapping $mapping, SeoUrlRouteConfig $config): ?string
     {
         try {
-            return trim($this->twig->render('template', $mapping->getSeoPathInfoContext()));
-        } catch (\Throwable $error) {
+            return trim($twig->render('template', $mapping->getSeoPathInfoContext()));
+        } catch (Error $error) {
+            $this->logger->critical('Error received on rendering SEO URL template', [
+                'exception' => $error,
```

Comment:
log the `$mapping`. Otherwise this will be pretty useless

Reply:
Good hint!